pipeline/pipeline.go

```diff
@@ -0,0 +1,28 @@
+package pipeline
+
+import (
+	"os/exec"
+
+	csvParser "github.com/m-lab/clinic2019/CsvParser"
+	"github.com/m-lab/clinic2019/incident_viewer_demo/incident"
+)
+
+type IncidentProducer interface {
+	FindIncidents()
+}
+
+type AnalyzerIncidents struct{}
+
+func (ai *AnalyzerIncidents) findIncidents() {
```

Comment:
@jacquigiese we can probably do something like this while running the gcs fuse mounting command

Reply:
yuhhh 🤙